docs/mozilla-traffic-cop.rst

```diff
@@ -56,20 +61,44 @@ An implementation might look like::
         }
     });
 
-    cop.init();
+    eddie.init();
 
 In the above example, the string "exp_firefox_new_headline" will be used as the
 cookie name should a variation be chosen.
 
 The test will have 3 variations, each targeting 25% of users.
 
+There is also an optional configuration value to specify how long the cookie
+associated with a visitor for an individual experiment will last:
+``cookieExpires``. This value must be a ``Number`` and represents the number of
+hours the cookie will last. If omitted, the cookie will only last for the
+current browsing session.
```

Comment:
This doesn't need to be part of this change, but i do wonder if defaulting to a session cookie is actually the best option for the majority of experiments. A session cookie could last for a long time potentially, especially if a user rarely restarts their browser which can be quite a common behavior. Defaulting to 24 hours could perhaps be a better limit?